src/app/client/src/app/app.component.ts

```diff
@@ -525,4 +526,33 @@ export class AppComponent implements OnInit, OnDestroy {
       }
     });
   }
+
+  async initializeCs() {
+    if (!CsModule.instance.isInitialised) {
+       // Singleton initialised or not
+        await CsModule.instance.init({
+          core: {
+              httpAdapter: 'HttpClientBrowserAdapter',
+              global: {
+                  channelId: this.channel, // required
+                  producerId: this.userService.appId, // required
+                  deviceId: this.fingerprintInfo // required
+              },
+              api: {
+                  host: document.location.origin, // default host
+                  authentication: {
+                      // userToken: string; // optional
+                      // bearerToken: string; // optional
+                  }
+              }
+          },
+          services: {
+              groupServiceConfig: {
+                apiPath: 'api/v1/group',
```

Comment:
apiPath is not correct check other apis which are calling learner service

Reply:
Modified with 'learner/v1/group'